src/snapred/backend/dao/request/FarmFreshIngredients.py

```diff
@@ -13,10 +13,11 @@ class FarmFreshIngredients(BaseModel):
     from these, the Sous Chef can make everything
     """
 
-    # NOTE this class ia a REQUEST object, despite its name
+    # NOTE this class is a REQUEST object, despite its name
     # Do NOT use inside ingredients for algorithms
 
     runNumber: str
+    version: Optional[str]
```

Comment:
Curious to why version is made into an optional attribute? Is this due to run numbers not having initialized state (and therefore no associated versions) when they are first reduced?

Reply:
**Prior to _acceptance_ by a user, all of the calibration results have no version.**  Personally, I would have preferred giving them a version as the latest result, and then just dropping the results if they are rejected.  However, this "no version" treatment is the way that Robert wanted to do it.  (Note that actually giving them a version, would allow the WNG and filename generation to be a bit more straightforward.  However, at this point, I think this is a minor issue and should wait for refactoring.)